docs/getting_started/features/flexible_services.md

```diff
@@ -1,11 +1,295 @@
+
+**Pre-requirement**: 
+
+- [Base features](/getting_started/features/base)
 
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
 |[stop_times.txt](/schedule/reference/#stop_timestxt)|`continuous_pickup`, `continuous_drop_off` |
 |[routes.txt](/schedule/reference/#routestxt)|`continuous_pickup`, `continuous_drop_off` |
+
+??? note "Sample Data"
+
+    <p style="font-size:16px">
+    The following sample shows two ways of representing Continuous Stop. 
+    </p>
+    
+    <p style="font-size:16px">
+    The first sample shows that pickups and dropoffs are allowed at any point along route `RA`.
+     </p>
+
+    <p style="font-size:16px">
+    The second sample shows that pickups and dropoffs are allowed between the third and fifth stops of trip `AWE1`, accomplished by assigning `continuous_pickup` and `continuous_drop_off` values to `stop_sequence=3` and `stop_sequence=4`.
+    </p>
+
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#routestxt"><b>routes.txt</b></a> <br>
+        </p>
+
+        | route_id | route_short_name | route_type | continuous_pickup | continuous_drop_off |
+        |----------|------------------|------------|-------------------|---------------------|
+        | RA       |               17 |          3 |                 0 |                   0 |
+
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#stop_timestxt"><b>stop_times.txt</b></a> <br>
+        </p>
+
+        | trip_id | arrival_time | departure_time | stop_id | stop_sequence | continuous_pickup | continuous_drop_off |
+        |---------|--------------|----------------|---------|---------------|-------------------|---------------------|
+        | AWE1    |      6:10:00 |        6:10:00 | TAS001  |             1 |                   |                     |
+        | AWE1    |      6:14:00 |        6:14:00 | TAS002  |             2 |                   |                     |
+        | AWE1    |      6:20:00 |        6:20:00 | TAS003  |             3 |                 0 |                   0 |
+        | AWE1    |      6:23:00 |        6:23:00 | TAS004  |             4 |                 0 |                   0 |
+        | AWE1    |      6:25:00 |        6:25:00 | TAS005  |             5 |                   |                     |
+
+##  Booking Rules
+
+Booking rules can be used to enable users to reserve a trip on a demand-responsive service. These rules outline the necessary prerequisites for successful bookings and provide contact information where users can make trip reservations. This should be used in conjunction with the On-demand services feature.
```

Comment:
....in conjunction with the **demand-responsive** services feature?

Reply:
Good catch! I'll replace it with the text: "This feature should be used in conjunction with `Predefined Routes With Deviation`, `Zone-based Demand Responsive Services` and `Fixed-Stops Demand Responsive Services features`, if such services require booking."